azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/ClientEntity.java

```diff
@@ -35,15 +35,15 @@ public String getClientId() {
         return this.clientId;
     }
 
-    boolean getIsClosed() {
+    public boolean getIsClosed() {
```

Comment:
>public [](start = 4, length = 6)

These 2 methods represent internal states and I think will need more work to expose them publicly.
for ex: getIsClosed() is the only method that should eventually be public and should return what isClosingOrClosed today returns - effectively it should indicate if close if raised or not.

Typically you shouldn't need these states to code against our client - as - the "caller" of the ClientEntity controls when the client is closed. Can you please specify why is this change needed ?

Reply:
I have an in-memory cache of EventHubClients. Whenever there's a problem with the EventHubClient you are closing the object and it can no-longer be usable.
I want to detect that recreate the client when that happens.